.circleci/scripts/binary_linux_test.sh

```diff
@@ -97,8 +97,17 @@ if [[ "$PACKAGE_TYPE" == conda ]]; then
   )
 elif [[ "$PACKAGE_TYPE" != libtorch ]]; then
   if [[ "\$BUILD_ENVIRONMENT" != *s390x* ]]; then
-    pip install "\$pkg" --index-url "https://download.pytorch.org/whl/\${CHANNEL}/${DESIRED_CUDA}"
-    retry pip install -q numpy protobuf typing-extensions
+    if [[ "$USE_SPLIT_BUILD" == "true" ]]; then
+      pkg="$(ls -1 /final_pkgs/torch_no_python* | sort |tail -1)"
+      pip install "\$pkg" --index-url "https://download.pytorch.org/whl/\${CHANNEL}/${DESIRED_CUDA}"
```

Comment:
- 1 pip command should be enough 
- The channel needs to be set to your package s3 path, now its set to: ``https://download.pytorch.org/whl/nightly/cu121``